### PR TITLE
Delete filter after recieved filter_deleted event

### PR DESCRIPTION
**Pull request by ehgmsdk20**
Changes:
- Delete filter after getting FILTER_DELETED event
- Delete filter if get REGISTER_FAILED error

Related issues:
- https://github.com/ainblockchain/ain-js/issues/88

### CODE REVIEW

**Comment by platfowner** [comment]
@ehgmsdk20 is this pr ready for review? if so, could you set the reviewer list?

**Comment by ehgmsdk20** [comment]
> @ehgmsdk20 is this pr ready for review? if so, could you set the reviewer list?

Oh, i forgot it. Now i added reviewers, thanks!

File: src/constants.ts [hunk 1]
```diff
@@ -1 +1,36 @@
 export const BLOCKCHAIN_PROTOCOL_VERSION = '1.0.0';
+
+export const EVENT_HANDLER_ERROR_CODE = {
```
Comment (platfowner) [inline src/constants.ts:3]: I think having two places for the full error code list (in the blockchain code and here) would increase the maintenance cost. what about just having here only those needed, i.e., FAILED_TO_REGISTER_FILTER?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: src/constants.ts (hunk 1 above)
Comment (ehgmsdk20) [reply src/constants.ts:3]: Oh, it would be nice. i will fix it

Comment (ehgmsdk20) [reply src/constants.ts]: Fixed, thanks!

**Comment by ehgmsdk20** [comment]
Thanks for review, now merging~

**Pull request merged**